bentoml/deployment/sagemaker/__init__.py

```diff
@@ -261,6 +259,57 @@ def init_sagemaker_project(sagemaker_project_dir, bento_path):
     return sagemaker_project_dir
 
 
+def _create_sagemaker_model(
+    sagemaker_client, bento_name, bento_version, ecr_image_path, api_name
+):
+    execution_role_arn = get_arn_role_from_current_aws_user()
+    model_name = create_sagemaker_model_name(bento_name, bento_version)
+
+    sagemaker_model_info = {
+        "ModelName": model_name,
+        "PrimaryContainer": {
+            "ContainerHostname": model_name,
+            "Image": ecr_image_path,
+            "Environment": {
+                "API_NAME": api_name,
+                "BENTO_SERVER_TIMEOUT": config().get('apiserver', 'default_timeout'),
+                "BENTO_SERVER_WORKERS": config().get(
+                    'apiserver', 'default_gunicorn_workers_count'
```

Comment:
this is configured on Yatai server but it's not where the workload is running, let's revisit this later, I believe this should be a user-provided parameter when creating sagemaker deployment, and if it's not there we simply don't set this env var, I believe in that case, we will use the bentoml.server.utils.get_bento_recommend_gunicorn_worker_count to get this value